app/app.go

```diff
@@ -569,34 +557,31 @@ func NewRegenApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 	return app
 }
 
+// TODO(Tyler): is below needed?
+
 // MakeCodecs constructs the *std.Codec and *codec.LegacyAmino instances used by
 // Regenapp. It is useful for tests and clients who do not want to construct the
 // full Regenapp
-func MakeCodecs() (codec.Codec, *codec.LegacyAmino) {
-	config := MakeEncodingConfig()
-	return config.Codec, config.Amino
-}
+//func MakeCodecs() (codec.Codec, *codec.LegacyAmino) {
+//	cfg := MakeEncodingConfig()
+//	return cfg.Codec, cfg.Amino
+//}
```

Comment:
We don't use this anywhere within our code base. Not sure if it would still be useful to external projects.

Reply:
I would say safe to remove given `MakeEncodingConfig` is public as well.

Reply:
i think i used it in the bridge.. or could have used it.. perhaps we can just leave it for the external projects